.github/workflows/release.yml

```diff
@@ -23,48 +27,39 @@ jobs:
       # Check out current repository
       - name: Fetch Sources
         uses: actions/checkout@v2
-        with:
-          ref: ${{ github.event.release.tag_name }}
+
+      # Set environment variables
+      - name: Export Properties
+        id: properties
+        shell: bash
+        run: |
+          PROPERTIES="$(./gradlew properties --console=plain -q)"
+          VERSION="$(echo "$PROPERTIES" | grep "^version:" | cut -f2- -d ' ')"
+          NAME="$(echo "$PROPERTIES" | grep "^pluginName_:" | cut -f2- -d ' ')"
+          ARTIFACT="${NAME}-${VERSION}.zip"
```

Comment:
This generated the filename that is expected to be created by the gradle buildPlugin. I have copied this from the pre-release section that I deleted